src/javascript/detectors/improper_access_control/improper_access_control.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// {fact rule=improper-access-control@v1.0 defects=1}
+var express = require('express')
+var helmet = require('helmet')
+var app = express()
+
+function improperAccessControlNoncompliant(){
+    app.use(
+        helmet.permittedCrossDomainPolicies({
+            // Noncompliant: permittedPolicies is set to 'false'.
+            permittedPolicies: false,
```

Comment:
From the docs:
> options.permittedPolicies is a string that must be "none", "master-only", "by-content-type", or "all". It defaults to "none".

Would it be better to use one of these values as noncompliant?

Reply:
Improved example.